django/thunderstore/community/views/community.py

```diff
@@ -0,0 +1,85 @@
+from django.conf import settings
+from django.db.models import BigIntegerField, Count, Sum
+from django.shortcuts import redirect
+from django.views import View
+from django.views.generic.list import ListView
+
+from thunderstore.cache.cache import CacheBustCondition
+from thunderstore.cache.pagination import CachedPaginator
+from thunderstore.community.models.community_site import CommunitySite
+from thunderstore.repository.models.package import Package
+
+# Should be divisible by 4 and 3
```

Comment:
I don't think this applies with the current layout? The original list supported showing 3 or 4 items per row, so this was used to ensure that each row was full. But at least the static image in this PR shows 5 items per row?

Reply:
Yeah, forgot to edit this. Fixd